dbms/src/Flash/Mpp/MPPTaskManager.cpp

```diff
@@ -31,11 +34,46 @@ extern const char random_task_manager_find_task_failure_failpoint[];
 extern const char pause_before_register_non_root_mpp_task[];
 } // namespace FailPoints
 
+namespace
+{
+// Print log for MPPTask which hasn't been removed for over 5 minutes.
```

Comment:
5 minutes is too short, I think it should be at least 30 minutes.

Reply:
> 5 minutes is too short, I think it should be at least 30 minutes.

okk